lab3/lab3-proj/src/Domain/ProgramInternalForm.java

```diff
@@ -0,0 +1,26 @@
+package Domain;
+
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/*
+public class ProgramInternalForm {
+    private List<Map.Entry<String,Integer>> pairList=new ArrayList<>();
+
+    public ProgramInternalForm() {
+    }
+
+
+
+
+    public void addInPIF(String token){
+        this.pairList.add(new AbstractMap.SimpleEntry<String,Integer>(token, 0));
```

Comment:
in PIF ar trebui sa adaugi o pereche <token, position> si ai 3 cazuri:

- daca token este identifier atunci trebuie sa adaugi <1, pos din st>
- daca token este constant atunci trebuie sa adaugi <0, pos din st>
- daca este altceva (separator, reserved word, operator) atunci trebuie sa adaugi <token, pos>